lib/common.h

```diff
@@ -8,9 +8,10 @@ using std::string;
 using std::vector;
 
 namespace common {
-
-    static const symbol S_SYS = symbol("EOS", 4);
+    static const symbol S_SYS = symbol("SYS", 4);
     static const symbol S_RAM = symbol("RAMCORE", 4);
+    static asset CPU(0'4000, S_SYS);
+    static asset NET(0'1000, S_SYS);
```

Comment:
Why are we setting these as `static`? I thought the plan was to eventually have these be dynamic arguments?

Reply:
Eventually it would be dynamic arguments, but for the initial version, these are static as the same quantity is used by createFreeAccounts and createJointAccounts.